modules/ROOT/pages/provisioning-nutanix.adoc

```diff
@@ -59,4 +59,96 @@ EOF
 
 == Launching a VM instance
 
-You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script".
+You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script". You can use the Nutanix Prism Central API to launch a VM in Nutanix:
+
+[source, bash]
+----
+CLUSTER_REFERENCE_NAME=<name of cluster to use>
+CLUSTER_REFERENCE_UUID=<uuid of cluster to use>
+IGNITION_CONFIG="$(cat <ign.config>) | base64"
+IMAGE_ID=$(curl -X POST --header "Content-Type: application/json" \
+         --header "Accept: application/json" \
+         --header "Authorization: Basic ${ENCODED_CREDS}" \
```

Comment:
Should we define `ENCODED_CREDS`, `API_HOST`, etc. again for anyone who skipped the previous section?